### ISSUE TITLE
[Accessibility- Script C 7.1] Some buttons don't contain a title

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2991]
**Environment:** 7.x

**Current behavior:**

- In the stream page, the buttons **"Send kudos"** , **"Filter feed"**  and the **three points button** near each comment (see attachement) have no label

<img width="127" height="71" alt="Image" src="https://github.com/user-attachments/assets/8f3af4c6-cb47-4b2e-ae13-0afbd7d65b75" />

- In the spaces submenus, the buttons **"Marked as read"** , **"Send a chat message"**, **"Send a kudos"** and **"Start a call"** have no label.

- In the left navigation menu, the **"Meeds"** link has an aria-label="See more"

**Expected behavior:**

Add aria-labels for the buttons above as:

- aria-label="Send kudos"
-  aria-label="Filter feed"
- aria-label= "Marked as read"
- aria-label="Send a chat message"
- aria-label ="Start a call"
- and for Meeds, add an aria-label="See more about Meeds"

![Image](https://github.com/user-attachments/assets/62812ba5-d4f2-4295-9c90-be283792623d)

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
